lib/src/HttpUtils.cc

```diff
@@ -745,4 +714,26 @@ std::pair<ContentType, const std::string_view> fileNameToContentTypeAndMime(
         return {CT_NONE, ""};
     return {CT_CUSTOM, it->second};
 }
+
+std::list<std::string_view> getFileExtensions(ContentType contentType)
```

Comment:
How about using `const std::vector<std::string_view> &` as the return type.

Reply:
`std::list` is implemented as a doubly-linked list, which means worse space complexity and worse locality compared to `std::vector`.  We should use `std::vector` unless we absolutely need to use it.

Reply:
Really? `std::unordered_map` is O(1) ?
I though it was O(N), that's why I used `std::map` instead.
For `std::vector`, you are right. I didn't take care.

I'm busy working on my media server, I have to deliver an pre-version asap.
So I'll do the changes in the PR next week.

Reply:
Yes, it's O(1), If fast lookup is required and the order of elements is not a concern, std::unordered_map might be the better choice.